app/project.properties

```diff
@@ -11,5 +11,5 @@
 split.density=false
 # Project target.
 target=Google Inc.:Google APIs:19
-android.library.reference.1=../libraries/mapballoons/android-mapviewballoons
```

Comment:
was this not working prior? I'd really like to prevent us from needing these kinds of no-op syntax updates wherever possible (the 3->2 is fine, since that'll get updated automatically, but the mixed filepath separators seem like they're gonna require this file to get updated again int he future.